raft/src/main/java/org/apache/kafka/raft/QuorumConfig.java

```diff
@@ -47,7 +47,7 @@
  * controller should be able to transition from standby to active without reloading all of
  * the metadata. The standby is a "hot" standby, not a "cold" one.
  */
-public class QuorumConfig {
+public class QuorumConfig extends AbstractConfig {
```

Comment:
Does it need to extend `AbstractConfig` if it's constructor accepts a `AbstractConfig` already?

Reply:
> Does it need to extend `AbstractConfig` if it's constructor accepts a `AbstractConfig` already?

Indeed, having it extend AbstractConfig is unnecessary. And I have removed it.   PTAL